spec/integration/runner/variant-reassignment-example-16.spec.js

```diff
@@ -0,0 +1,69 @@
+ * | slug: { en: "bike" }               | id: 1                                |                    | id: 2                                                      |
+ * | masterVariant: { sku: "red-bike" } | slug: { en: "bike" }                 |                    | slug: { en: "tshirt" }                                     |
+ * |                                    | masterVariant: { sku: "red-tshirt" } |                    | masterVariant: { sku: "red-bike" }                         |
+ * +------------------------------------+--------------------------------------+--------------------+------------------------------------------------------------+
+ * |                                    | Product:                             |                    | Product:                                                   |
+ * |                                    | id: 2                                |                    | id: 1                                                      |
+ * |                                    | slug: { en: "tshirt" }               |                    | slug: { en: "bike-${timestamp}", ctsd: "${timestamp}" }    |
+ * |                                    | masterVariant: { sku: "red-bike" }   |                    | masterVariant: { sku: "red-tshirt" }                       |
+ * +------------------------------------+--------------------------------------+--------------------+------------------------------------------------------------+
+ */
+/* eslint-enable max-len */
+describe('Variant reassignment', () => {
+  let ctpClient
+  let product1
+  let product2
+  const logger = utils.createLogger(__filename)
+  const product1Sku = 'red-tshirt'
+  const product2Sku = 'red-bike'
+
+  before(async () => {
+    ctpClient = await utils.createClient()
+    const products = await utils.createCtpProducts([[product1Sku], [product2Sku]], ctpClient,
+      (pD) => {
+        if (pD.masterVariant.sku === product1Sku)
+          pD.slug = { en: 'bike' }
+        else
+          pD.slug = { en: 'tshirt' }
+      })
+    product1 = products.find(product => product.masterVariant.sku === product1Sku)
+    product2 = products.find(product => product.masterVariant.sku === product2Sku)
+  })
+
+  after(() =>
+    utils.deleteResourcesAll(ctpClient, logger)
+  )
+
+  it('only anonymize product', async () => {
+    const testStartTstp = new Date()
+    const reassignment = new VariantReassignment(ctpClient, logger, {}, ['brandId'])
+    await reassignment.execute([{
+      productType: {
+        id: product1.productType.id
+      },
+      name: product1.name,
+      slug: product1.slug,
+      masterVariant: product2.masterVariant
+    }], [product2])
+    const { body: { results } } = await utils.getProductsBySkus([product1Sku, product2Sku],
+      ctpClient)
+    expect(results).to.have.lengthOf(2)
+    const notUpdatedProduct = results.find(product => product.masterVariant.sku === product2Sku)
```

Comment:
Could you add also `expect` here that the first product does not have `ctsd` slug and some more expectations?

Reply:
I already have the expectation that first product is not changed, so I think it's enough.